com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Added a supported MSAA samples count check, so the actual supported MSAA samples count value can be assigned to RenderTexture descriptors.
+- Added a terrain compatiblity shader tag for validating whether user overridden materials support terrain rendering functionalities.
```

Comment:
See the comment in HDRP (if that comment is correct, use it here as well).

Reply:
Changed to match revisions made to URP's ChangeLog